adhocracy4/datetimefield/templates/a4datetimefield/datetime_input.html

```diff
@@ -0,0 +1,9 @@
+<div class="l-col-2 widget--datetimeinput__date">
+    {{ date }}
+</div>
+<div class="l-col-2">
```

Comment:
In order to be reusable this should probably have a class like `widget--datetime__time`? I have not that much experience with reusing styling, maybe @xi has some advise here?

Reply:
I see three options

1. Use specific, non-reusable selectors like `a4-datetime__time`. CSS needs to be done in the individual projects, possible by using `@extend` with existing selectors.
2. Use roughly the same selectors and styling in all projects.
3. Overwrite the template in each project. The template here is merely an example for documentation purposes.

We are currently doing a bit of all of them (which I think is totally fine btw). I tend to prefer (3), but that doesn't work for react components.